src/environments/environment.stage.ts

```diff
@@ -0,0 +1,9 @@
+export const environment = {
+    production: true,
+    relayUrl: 'https://staging.wirlernenonline.de/relay/graphql',
+    editorBackendUrl: 'https://staging.wirlernenonline.de/editor/graphql',
```

Comment:
The format has slightly changed, not including the `/graphql` anymore.
```suggestion
    relayUrl: 'https://staging.wirlernenonline.de/relay',
    editorBackendUrl: 'https://staging.wirlernenonline.de/editor',
```

Reply:
done